Demo/Demo/Models/AccessTokenResponse.swift

```diff
@@ -0,0 +1,8 @@
+public struct AccessTokenResponse: Codable {
```

Comment:
Same question for this file - just want to make sure it's only supposed to be in the demo app